src/main/java/com/bnkrll/controller/SessionController.java

```diff
@@ -35,4 +35,18 @@ public List<Session> getPreviousSessions(
                     int last) {
         return sessionRepository.getLastSessions(last);
     }
+
+    @PatchMapping
+    public ResponseEntity<Session> updateSession(@RequestBody Session session){
```

Comment:
maybe you could scope the update to only update the buy-in. These thinner layers can help tell the story of your domain more easily. It's also more secure.

Reply:
Sure, update the controller name to updateBuyInOfSession

Reply:
not just the name. Send in only the id and new buy-in amount.